Arsany

arsanyy opened the pull request:
I created two empty files

arsanyy committed:
Create test1.ipynb

arsanyy committed:
create text file

karimamd reviewed (changes requested):
please include some code in the files

arsanyy committed:
data science poject

This hotel dataset should be an easy one. I decided to work on it due to the tight deadline.

arsanyy committed:
hopefully All Done

this is the final version I hope XD all data should be clean and ready for reporting

arsanyy committed:
last edit hopfully #2
